packages/ui/package.json

```diff
@@ -19,6 +19,7 @@
     "react-dom": "^17.0.1",
     "react-scripts": "4.0.1",
     "styled-components": "^5.2.1",
+    "styled-icons": "^10.29.0",
```

Comment:
If we use these only in the stories then this will be a `devDependency` dont you think?